services/galley/test/integration/API/MLS.hs

```diff
@@ -406,9 +406,21 @@ testAddUserWithProteusClients = do
 
 testAddUserPartial :: TestM ()
 testAddUserPartial = do
-  (creator, commit) <- withSystemTempDirectory "mls" $ \tmp -> do
+  (creator, commit) <- withSystemTempDirectory "mls" $ \_tmp -> do
+    let tmp = "/tmp/mls"
```

Comment:
This manual assignment of `tmp` looks like a leftover from debugging.

Reply:
Fixed